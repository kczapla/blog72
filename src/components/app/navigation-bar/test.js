import React from 'react'
import renderer from 'react-test-renderer'

import NavigationBar from './index'

jest.mock('react-router-dom', () => {
  return {
    __esModule: true,
    Link: function Link () { return <a/> }
  }
})

describe('navigation bar', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(<NavigationBar/>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
