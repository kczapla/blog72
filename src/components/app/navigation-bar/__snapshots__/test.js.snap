// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`navigation bar renders correctly 1`] = `
<nav
  className="sc-gsTCUz cYoknp"
>
  <div
    className="sc-dlfnbm ldmZRN"
  >
    <a />
    <a />
    <a />
  </div>
</nav>
`;
